### PR TITLE
[docs](unique-key-merge-on-write) add document for unique key merge o…

### PR DESCRIPTION
**Pull request by zhannngchen**
…n write

# Proposed changes

Issue Number: close #xxx

## Problem summary

Update english version later

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/data-table/data-model.md [hunk 1]
```diff
@@ -308,7 +310,54 @@ PROPERTIES (
...
+| user_id       | BIGINT       |                 | 用户id       |
+| username      | VARCHAR(50)  |                 | 用户昵称     |
+| city          | VARCHAR(20)  | NONE            | 用户所在城市 |
+| age           | SMALLINT     | NONE            | 用户年龄     |
+| sex           | TINYINT      | NONE            | 用户性别     |
+| phone         | LARGEINT     | NONE            | 用户电话     |
+| address       | VARCHAR(500) | NONE            | 用户住址     |
+| register_time | DATETIME     | NONE            | 用户注册时间 |
+
+在开启了写时合并选项的Unique表上，数据在导入阶段就会去将被覆盖和被更新的数据进行标记删除，同时将新的数据写入新的文件。在查询的时候，所有被标记删除的数据都会在文件级别被过滤掉，读取出来的数据就都是最新的数据，消除掉了读时合并中的数据聚合过程，并且能够在很多情况下支持多种谓词的下推。因此在许多场景都能带来比较大的性能提升，尤其是在有聚合查询的情况下。
```
Comment (morningman) [inline docs/zh-CN/docs/data-table/data-model.md:360]: 1. 历史表如何处理？是否能转换成新的格式？
2. 原有unique key中的delete flag 和sequence col 如何处理？
3. 对应的英文文档修改

Comment (zhannngchen) [reply docs/zh-CN/docs/data-table/data-model.md:360]: updated

**Review by hf200012 — APPROVED** [review]
LGTM

File: docs/en/docs/data-table/data-model.md [hunk 2]
```diff
@@ -303,7 +305,60 @@ PROPERTIES (
...
 ```
 
-That is to say, Uniq model can be completely replaced by REPLACE in aggregation model. Its internal implementation and data storage are exactly the same. No further examples will be given here.
+That is to say, the merge-on-read implementation of the Unique model can be completely replaced by REPLACE in aggregation model. Its internal implementation and data storage are exactly the same. No further examples will be given here.
+
+### Merge on write (introduced from version 1.2)
+
+The merge-on-write implementation of the Unique model is completely different from the aggregation model. The query performance is closer to the duplicate model. Compared with the aggregation model, it has a better query performance.
+
+In version 1.2, as a new feature, merge-on-write is disabled by default, and users can enable it by adding the following property
```
Comment (dataroaring) [inline docs/en/docs/data-table/data-model.md:314]: In version 1.2, as a new feature, merge-on-write is disabled by default, and users can enable it by adding the following property in create table statement?

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**